src/typechecker/FStar.TypeChecker.TcTerm.fs

```diff
@@ -1085,14 +1085,20 @@ and tc_value env (e:term) : term
     //FIXME: Check context inclusion?
     value_check_expected_typ env e (Inl (SS.subst' s u.ctx_uvar_typ)) Env.trivial_guard
 
-  | Tm_unknown -> //only occurs where type annotations are missing in source programs
+  //only occurs where type annotations are missing in source programs
+  //or the program has explicit _ for missing terms
+  | Tm_unknown ->
     let r = Env.get_range env in
     let t, _, g0 =
         match Env.expected_typ env with
-        | None ->  let k, u = U.type_u () in
-                   TcUtil.new_implicit_var "type of user-provided implicit term" r env k
+        | None ->
+          let k, u = U.type_u () in
+          TcUtil.new_implicit_var "type of user-provided implicit term" r env k
         | Some t -> t, [], Env.trivial_guard in
-    let e, _, g1 = TcUtil.new_implicit_var ("user-provided implicit term at " ^ (Range.string_of_range r)) r env t in
+    
+    let e, _, g1 = TcUtil.new_implicit_var
+          ("user-provided implicit term at " ^ (Range.string_of_range r))
+          r env t in
```

Comment:
Looks to be only formatting and comment change here. Am I missing something

Reply:
Yeah it is just formatting, the line was too long.